test/Directory.Build.props

```diff
@@ -2,7 +2,7 @@
 <Project>
   <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
   <PropertyGroup>
-    <TargetFramework>net8.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
why is this set to 6.0? Aren't we moving to 8.0?

Reply:
The `src/` projects are still `net6.0` but the tests got bumped to `8.0` which was causing some problems. We actually are not moving the projects much, just Arcade